Source/Compression.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  Compression.swift
+//  Starscream
+//
+//  Created by Joseph Ross on 5/23/17.
+//  Copyright © 2017 Vluxe. All rights reserved.
+//
+
```

Comment:
Link to the spec here so that later if there is a bug one can quickly find the reference to follow: https://tools.ietf.org/html/rfc7692